src/api/api.h

```diff
@@ -0,0 +1,12 @@
+#ifndef OIDC_AGENT_API_H
+#define OIDC_AGENT_API_H
+
```

Comment:
For compatibility reasons we must preserve a single `api.h` files that provides everything needed.